wp-graphql.php

```diff
@@ -196,6 +196,8 @@ private function includes() {
 		 */
 		private function actions() {
 			// @placeholder where actions can be added throughout. This will be useful for mutations
```

Comment:
we can get rid of this placeholder comment now that this method is in use and not actually a placeholder